nixos/modules/system/etc/etc.nix

```diff
@@ -62,6 +62,21 @@ let
     ]) etc'}
   '';
 
+  etcHardlinks = filter (f: f.mode != "symlink") etc';
+
+  build-composefs-dump = pkgs.runCommand "build-composefs-dump.py"
```

Comment:
Can we use here `writePython3` instead?

Reply:
Yes, but writePython3 runs flake8 in the `checkPhase`. Since I have been convinced that we should rather not do any checks (because they pull in unnecessary dependencies---a pain I'm feeling as well especially when re-building foundational software like systemd) this doesn't make sense to my anymore either.